basement_bot/admin.py

```diff
@@ -304,3 +305,68 @@ async def _bot_data(self, ctx):
         embed.set_thumbnail(url=self.bot.user.avatar_url)
 
         await self.bot.tagged_response(ctx, embed=embed)
+
+    @decorate.with_typing
+    @commands.command(name="issue", aliases=["ish", "botish", "botissue"])
+    async def issue(self, ctx, title: str, description: str):
+        """Creates an issue in the bot's Github Repo
+
+        This is a command and should be accessed via Discord.
+
+        parameters:
+            ctx (discord.ext.Context): the context object for the calling message
+            title: the title of the issue
+            description: the description of the issue
+        """
+        GITHUB_API_BASE_URL = "https://api.github.com"
```

Comment:
you can add these constants to the class level where `ADMIN_ONLY = True`, then reference them as `self.GITHUB_API_BASE_URL`

Reply:
do you want it for the icon too?

Reply:
Yeah, not a big deal but I like to keep all the constants at the class level